Makefile

```diff
@@ -95,13 +100,12 @@ run: generate fmt vet
 		KUBEVIRT_CLIENT_GO_SCHEME_REGISTRATION_VERSION=v1 go run ./cmd/forklift-controller/main.go
 
 # Install CRDs into a cluster
-install: manifests
-	kubectl apply -f config/crds
+install: manifests kubectl
+	$(KUBECTL) apply -k operator/config/crds
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-deploy: manifests
-	kubectl apply -f config/crds
-	kustomize build config/default | kubectl apply -f -
+deploy: manifests kubectl
+	$(KUBECTL) apply -k operator/config/default
```

Comment:
should be `kustomize build operator/config/default | $(KUBECTL) apply -f `, no?

Reply:
That's equivalent to `$(KUBECTL) apply -k operator/config/default`

From `kubectl apply -h`:
```
    -k, --kustomize='':
	Process a kustomization directory. This flag can't be used together with -f or -R.
```